uktrade/data-hub-frontend
# Show Lead Adviser.

## Description of change

Prototype - https://projects.invisionapp.com/share/D6U38OMCE3G#/screens/388971543
Trello - https://trello.com/c/CWbefPda/974-state-2-on-new-tab

This is the second stage around adding a Lead Adviser to a company. You should now be able to add yourself as a Lead adviser then see the adviser details in a table within the 'Lead adviser' tab

## Test instructions
To see this change you will need two set things:

1. Feature flag `lead_advisers` must be active
2. You must post yourself as the lead adviser, see API docs - https://api.datahub.dev.uktrade.io/docs

You will have to post yourself via the likes of Postman as we have not built the form submit yet.
 
## Screenshots
![Screenshot 2019-11-11 at 16 25 29](https://user-images.githubusercontent.com/10154302/68604007-680ab780-04a1-11ea-901a-7eb4eb4a5755.png)

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
